source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ViewController.m

```diff
@@ -107,11 +142,48 @@ - (void)update:(NSString *) jsonStr
         adcVc.acrActionDelegate = self;
         if(self.curView)
             [self.curView removeFromSuperview];
+        else
+        {
+            self.scrView = [[UIScrollView alloc] initWithFrame:CGRectMake(0,0,0,0)];
+            self.scrView.showsHorizontalScrollIndicator = YES;
+        }
         self.curView = adcVc.view;
-        self.curView.frame = CGRectMake(20, 250, 300, 1250);
+        self.scrView.translatesAutoresizingMaskIntoConstraints = NO;
+        NSLog(@"ContentView H = %f", self.curView.frame.size.height);
+        NSLog(@"ContentView W = %f", self.curView.frame.size.width);
```

Comment:
These logging statements, and the others, look like they're for debugging and can be removed?

Reply:
Thanks for catching this.

---
In reply to: [143580818](https://github.com/Microsoft/AdaptiveCards/pull/760#discussion_r143580818) [](ancestors = 143580818)